arkane/encorr/corr.py

```diff
@@ -42,20 +42,31 @@
 
 def get_energy_correction(model_chemistry, atoms, bonds, coords, nums, multiplicity=1,
                           atom_energies=None, apply_atom_corrections=True,
-                          apply_bacs=False, bac_type='p'):
+                          apply_bac=False, bac_type='p'):
     """
     Calculate a correction to the electronic energy obtained from a
     quantum chemistry calculation at a given model chemistry such that
     it is consistent with the normal gas-phase reference states.
     Optionally, correct the energy using bond additivity corrections.
+
+    model_chemistry: The model chemistry, typically specified as method/basis.
```

Comment:
You can format the arguments to get nice styling when we compile the API documentation. Minimum would be adding the `Args` keyword and indenting:
```
Args:
    model_chemistry: The model chemistry, typically specified as method/basis.
    ...
```

See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for even more details.

Reply:
Good to know!